WORKSPACE

```diff
@@ -220,6 +221,14 @@ pypi_repository(
     version = "3.4.0",
 )
 
+pypi_wheel_repository(
```

Comment:
Ack, thank your for the details.

@ArchangelX360 you might be able to compile the project :p